Cubical/Foundations/Equiv/HalfAdjoint.agda

```diff
@@ -144,3 +149,15 @@ congIso {x = x} {y} e = goal
                    ; (j = i0) → Iso.leftInv e x (i ∨ k)
                    ; (j = i1) → Iso.leftInv e y (i ∨ k) })
           (Iso.leftInv e (p j) i)
+
+invCongFunct : {x : A} (e : Iso A B) (p : Iso.fun e x ≡ Iso.fun e x) (q : Iso.fun e x ≡ Iso.fun e x)
```

Comment:
Yikes, this had a scary proof! Can't one do it by `elimIso` (https://github.com/agda/cubical/blob/master/Cubical/Foundations/Univalence.agda#L287) instead?

I guess one potential problem with that lemma is that it's using univalence... But maybe that's still simpler this proof? (which might be using univalence somewhere anyway...)

Reply:
Hmm, I'm a bit scared of this proof. But I realised it was super easy to prove with J. This doesn't even rely on the fact that we have an Iso, so that should be much better for computations.

Reply:
I'm confused, what are you using J on to prove it? 

As far as I can tell the proof is still scary btw...